internal/resource/url.go

```diff
@@ -294,9 +294,6 @@ func (f *Fetcher) fetchFromHTTP(u url.URL, dest io.Writer, opts FetchOptions) er
 // FetchFromDataURL writes the data stored in the dataurl u into dest, returning
 // an error if one is encountered.
 func (f *Fetcher) fetchFromDataURL(u url.URL, dest io.Writer, opts FetchOptions) error {
-	if opts.Compression != "" {
-		return ErrCompressionUnsupported
```

Comment:
Would there be any harm in dropping this from `fetchFromS3` as well? If not we could also drop the error message declaration.

Reply:
The S3 downloader fetches chunks in parallel and writes them out-of-order, so that would require code changes to support a second decompression pass.